official/utils/accelerator/tpu.py

```diff
@@ -105,7 +101,7 @@ def embedding_matmul(embedding_table, values, mask, name='embedding_matmul'):
   """
 
   with ops.name_scope(name):
-    n_embeddings, embedding_dim = embedding_table.get_shape().as_list()
+    n_embeddings, embedding_dim = embedding_table.get_shape().as_list()  # pylint: disable=unused-variable
```

Comment:
If embedding_dim is not used, can we just replace it with _?

Reply:
Sure. I just had it there for readability, but I can just as easily see _ as clearer.